dev-docs/bidders/cwire.md

```diff
@@ -5,7 +5,7 @@ description: C-WIRE Prebid Bidder Adapter
 pbjs: true
 pbs: true
 biddercode: cwire
-tcfeu_supported: false
+tcfeu_supported: true
```

Comment:
In order to claim `tcfeu_supported` you need a `gvl_id` . Otherwise the consent cannot be validated.

Reply:
The id is 1081. Where should we provide it?

Reply:
Hey @muuki88 
Thanks for the review. Do you need anything more from us?

Reply:
Is this also present in your prebid adapter?

Reply:
PR created for the adapter:
https://github.com/prebid/Prebid.js/pull/11181